pyoverkiz/client.py

```diff
@@ -129,6 +131,12 @@ def __init__(
 
         self.session = session if session else ClientSession()
 
+        self.api_type = (
```

Comment:
I'm not a big fan of this style.

Reply:
What would you prefer? Just a regular if/else statement?